code/modules/mob/living/carbon/human/species_types/furry/moth.dm

```diff
@@ -27,8 +27,8 @@
 		OFFSET_NECK_F = list(0,0), OFFSET_MOUTH_F = list(0,0), OFFSET_PANTS_F = list(0,0), \
 		OFFSET_SHIRT_F = list(0,0), OFFSET_ARMOR_F = list(0,0), OFFSET_UNDIES_F = list(0,0)
 		)
-	specstats = list("strength" = 0, "perception" = 1, "intelligence" = -1, "constitution" = 0, "endurance" = 1, "speed" = -1, "fortune" = 0)
-	specstats_f = list("strength" = -1, "perception" = 0, "intelligence" = 2, "constitution" = -1, "endurance" = 0, "speed" = 1, "fortune" = 0)
+	specstats = list("strength" = 0, "perception" = 0, "intelligence" = 0, "constitution" = -2, "endurance" = 0, "speed" = 2, "fortune" = 0)
+	specstats_f = list("strength" = 0, "perception" = 0, "intelligence" = 0, "constitution" = -2, "endurance" = 0, "speed" = 2, "fortune" = 0)
```

Comment:
maybe make them have -1 strength -1 end and +2 int/+1 int and +1 per? very boring stats